### ISSUE TITLE
Need to nail down the SD.ini location

### ISSUE THREAD

**Issue by danlyke** [mpogue2/SquareDesk#379]
Nigel is having troubles on the Mac using the SD pane because the sd.ini is getting created in some weird location, possibly the shared folder.

**Comment by mpogue2** [comment]
@danlyke, there is no sd.ini file anywhere on my Mac, and SD-in-SquareDesk works fine, so I'm not sure what the issue is here that Nigel is having....

**Comment by mpogue2** [comment]
@danlyke Did this get resolved?

**Comment by danlyke** [comment]
No, and I haven't gotten back around to re-looking at the SD stuff...

**Comment by mpogue2** [comment]
I'm pretty sure that the sd.ini file is no longer present in our build, nor do we need it.  Abbreviations (a typical use for sd.ini) are in abbrevs.txt now, handled completely outside of sd.  

So, closing this one.
